packages/components/select/src/select.js

```diff
@@ -29,6 +48,8 @@ export class TSSelect extends TSElement {
 			selected: { type: Array, reflect: true },
 			/** Default placeholder when there is no selection. */
 			placeholder: { type: String, reflect: true },
+			/** Translated messages for the user locale */
```

Comment:
nit, just ranting, we should think of a way to include the translation default object in the readme, by modifying the update-readme script 🤔 
Although at first we should come to an agreement in the guideline about translations.

Reply:
I agree, we should provide the default object and available strings (with description where they are used) in the README, but we should have a guideline first to have it in consistent way (plus be able to automate it with scripts).